lib/reservation.rb

```diff
@@ -0,0 +1,43 @@
+require "date"
+require "pry"
+
+class Reservation
+  attr_accessor :reservation_id, :check_in, :check_out, :room, :is_hotel_blocker
+
+  def initialize(reservation_id, check_in: nil, check_out: nil, room: nil, is_hotel_blocker: false)
```

Comment:
I would like most of these to be mandatory instead of being set after initializing the instance.

```suggestion
  def initialize(reservation_id, check_in:, check_out:, room:, is_hotel_blocker: false)
```

Right now it's possible to create a reservation with no room or date range associated with it.